simpson.cpp

```diff
@@ -0,0 +1,45 @@
+#include<stdio.h>
+#include<conio.h>
```

Comment:
Don't use this. Reformat this file using clang-format with 4 spaces. Place this file into the correct directory.

Reply:
@arnavb This is my first time contributing here. So, could you help regarding the correct directory?